matrix/common/aws/redshift_handler.py

```diff
@@ -50,3 +50,12 @@ def transaction(self, queries: typing.List[str]):
             cursor.execute(query)
         conn.commit()
         conn.close()
+
+    def run_query_and_return_results(self, query):
+        conn = pg.connect(self.database_uri)
+        cursor = conn.cursor()
+        cursor.execute(query)
+        results = cursor.fetchall()
```

Comment:
Can this be added to `transaction` as an optional param instead?

Reply:
Sounds good